frontend/apps/marketing/src/components/carousels/imageCarousel/ImageCarousel.tsx

```diff
@@ -0,0 +1,60 @@
+'use client';
+
+import '@code-dot-org/component-library/carousel/index.css';
+import React, {useMemo} from 'react';
+
+import DSCOCarousel, {
+  CarouselProps,
+} from '@code-dot-org/component-library/carousel';
+
+import Image from '@/components/image';
+import {ExperienceAsset} from '@/types/contentful/ExperienceAsset';
+
+export type ImageCarouselProps = {
+  /** Carousel content w/ fields from Contentful */
+  slides: ExperienceAsset[];
+  /** Number of slides to show at once */
+  slidesPerView?: Extract<CarouselProps['slidesPerView'], number>;
+};
+
+const ImageCarousel: React.FC<ImageCarouselProps> = ({
+  slides,
+  slidesPerView,
+}) => {
+  if (!slides) {
+    return (
+      <div style={{color: 'var(--text-neutral-primary)'}}>
+        <em>
+          <strong>🖼️ Image carousel placeholder.</strong> Please add a
+          "Carousel" content type entry in the Content sidebar, save, and open
+          the preview tab to see the carousel in action.
+        </em>
+      </div>
+    );
+  }
+
+  const slidesData = useMemo(
+    () =>
+      slides.filter(Boolean).map(({fields}) => {
+        const {title, file} = fields;
+
+        return {
+          id: title,
+          slide: <Image src={file?.url} altText="" />,
```

Comment:
1. Does Contentful provide any altText we can use? 
2. If the `file` doesn't exist, it'd show a broken image. Should we remove the slide instead?

Reply:
1. Yep! We can use the `description` field. I wasn't sure if we want alt text, but I'll update it use it. 
2. Good call, updated and tested in Contentful by deleting an image from the content entry.